cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java

```diff
@@ -157,6 +157,7 @@ protected static TestManager getTestManager() {
    * @param applicationClz The application class
    * @return An {@link co.cask.cdap.test.ApplicationManager} to manage the deployed application.
    */
+  @Deprecated
```

Comment:
All of them already have it in the comments.
For instance:
`* @deprecated Use {@link TestManager#deployApplication(Class, java.io.File...)} from {@link #getTestManager()}.`